pyproject.toml

```diff
@@ -36,7 +38,8 @@ mypy = "^1.10.1"
 hypothesis = "^6.105.1"
 graphviz = "^0.20"
 pre-commit = "^3.7.1"
-guppylang = "^0.6.0"
+# guppylang = "^0.6.0"
+guppylang = { git = "https://github.com/CQCL/guppylang", rev = "3da3936" }
```

Comment:
needed for hugr-py compatibility

Reply:
This will prevent us from releasing new tket2-py versions. Can we do a guppy release soon?